src/styles/index.scss

```diff
@@ -1,3 +1,134 @@
 body {
   margin: 0;
 }
+
+$bodyFont: Roboto, sans-serif;
+$bodyColor: rgba(6, 11, 53, 1);
+
+.main {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  height: 408px;
+  width: 200px;
+
+  &__img {
+    height: 134px;
+    width: 160px;
+
+    margin-top: 33px;
+    margin-bottom: 32px;
```

Comment:
It's better to choose one direction and set just top or bottom margins to avoid margin collapse